queries/eds/highlights.scm

```diff
@@ -0,0 +1,63 @@
+["="] @punctuation.delimeter
+
+(
+ (section_name) @variable.builtin
+ (#any-of? @variable.builtin "[FileInfo]" "[DeviceInfo]" "[DummyUsage]" "[MandatoryObjects]" "[OptionalObjects]")
+)
```

Comment:
I can be wrong and maybe it's intended to have section_name include the brackets, but extracting the brackets to the outer node which is then followed by the section_name node of just text could be nicer (not an issue here)